spackbot/__main__.py

```diff
@@ -31,8 +32,12 @@
 APP_IDENTIFIER = os.environ.get("GITHUB_APP_IDENTIFIER")
 REQUESTER = os.environ.get("GITHUB_APP_REQUESTER")
 
+# If we are given a file, load into variable
+if os.path.exists(PRIVATE_KEY):
+    with open(PRIVATE_KEY, 'r') as handle:
+        PRIVATE_KEY = handle.read()
```

Comment:
Can you merge this with the `fix_private_key()` method since all of that is about handling the many ways we can consume private keys?  Also I think if the bot ever becomes concurrent for whatever reason, we'll avoid potentially confusing effects of doing I/O on every import. Better to do that once in `main()` than once per process.